### PR TITLE
feat: Add API for obtaining list of workflows via workflow id, along with Titles

### PR DESCRIPTION
**Pull request by GrantMurray**
This should unblock Gabby.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -57,17 +59,25 @@ function setupGets(database) {
...
     const simulationReport = await database.collection('SimulationReport').findOne({PrintJobID: printJob._id, WorkflowID: workflowDoc._id});
-    // If no simulationReport is returned
-    if (!simulationReport) {
-      reply.code(404).send("Simulation report not found");
+    if (!simulationReport) reply.code(404).send("Simulation report not found");
+    else reply.code(200).send({PrintJob: printJob, SimulationReport: simulationReport});
+  });
+
+  fastify.get('/getWorkflowList', async (request, reply) => {
+    const workflowDocs = await database.collection('Workflow').find();
```
Comment (Caleb-Biggs) [inline server.js:71]: add .toArray() after find here

File: server.js [hunk 2]
```diff
@@ -57,17 +59,25 @@ function setupGets(database) {
...
+    if (!workflowDocs) {
+      reply.code(404).send("WorkflowDocs not found");
       return;
     }
-    // Else
-    reply.code(200).send({PrintJob: printJob, SimulationReport: simulationReport});
+    const workflowList = [];
+    for await (const doc of workflowDocs) {
```
Comment (Caleb-Biggs) [inline server.js:77]: This becomes unnecessary with .toArray()

**Review by Caleb-Biggs — APPROVED** [review]

File: test/integration/server.test.js [hunk 3]
```diff
@@ -76,6 +76,16 @@ test('POST /query', async (t) => {
...
 });
 
+test('GET /getWorkflowList', async (t) => {
+    const response = await fastify.inject({
+        method: 'GET',
+        url: '/getWorkflowList'
+    });
+    assert.strictEqual(response.statusCode, 200);
+    const payloadList = response.payload;
+    console.log("Workflows: ", payloadList);
```
Comment (amychisholm03) [inline test/integration/server.test.js:86]: I'm curious if this payloadList is actually a list of workflows -- I'm assuming you tested locally?

**Review by amychisholm03 — APPROVED** [review]
LGTM!

**Pull request merged**